### PR TITLE
Add F# interactive instructions

### PR DESCRIPTION
**Pull request by loic-sharma**
Screenshot:

image

Addresses https://github.com/NuGet/Engineering/issues/3527

/cc @cartermp @jonsequitur

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -91,7 +91,18 @@
...
+            },
+
+            new PackageManagerViewModel("F# Interactive")
+            {
+                Id = "fsharp-interactive",
+                CommandPrefix = "> ",
+                InstallPackageCommand = string.Format("#r \"nuget: {0}, {1}\"", Model.Id, Model.Version),
+                AlertLevel = AlertLevel.Info,
+                AlertMessage = string.Format(
+                    "For F# interactive projects that support <a href=\"{0}\">#r syntax</a>, copy this into the project file to reference the package.",
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml:103]: I believe that `project file` here is incorrect, I will confirm with the F# team:

```suggestion
                    "For F# interactive projects that support <a href=\"{0}\">#r syntax</a>, copy this into the source file to reference the package.",
```

**Comment by joelverhagen** [comment]
Can you put a screenshot in the description?

**Comment by loic-sharma** [comment]
@joelverhagen Added!

**Review by joelverhagen — APPROVED** [review]
Make sure it looks at least usable/reasonable at smaller resolutions or windows sizes then :shipit:

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 1 above)
Comment (baronfel) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml:103]: (not F# team but active community member) This is correct terminology, project files aren't used in interactive contexts.

**Review by zhhyu — APPROVED** [review]

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 1 above)
Comment (cartermp) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml:103]: Seconded @baronel's note here

Comment (cartermp) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml:103]: Slightly revised proposed wording:

> For F# scripts that support #r syntax, copy this into the source code to reference the package.

Comment (loic-sharma) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Great, thanks everyone for the feedback! I've updated the text accordingly

**Comment by loic-sharma** [comment]
@joelverhagen I verified this is reasonably usable in a mobile view. Sadly, the tabs wrap onto a new line:

image

This is existing behavior though on nuget.org:

image

I don't plan to fix this as part of this change.

**Comment by joelverhagen** [comment]
I've noticed this wrapping before and frankly, it's not that bad. Perhaps we could use a bit of polish there but it's still quite usable. The command itself wraps nicely so I'm happy with that if you are :)

**Pull request merged**